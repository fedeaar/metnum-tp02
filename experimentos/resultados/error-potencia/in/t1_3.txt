4.858292912650735502e+02 -6.335084886073239829e+00 -8.072867718038598639e+01 2.469966596815418569e+01 -1.097839013241607660e+02 -6.852810741588938015e+01 1.097972658845433003e+01 6.370785951675750347e+00 1.740591824716803160e+01 4.935306545000331724e+00 -1.213444492077767904e+02 -6.980803836514644445e+01 9.955283468031898841e-02 -1.537067629831232978e+02 -5.002806584102325616e+01 -4.432063532967829644e+01 -1.857468719893926448e+02 -7.120919595547843528e+01 -2.072681875815117678e+02 -3.843893266686814769e+01
-6.335084886073239829e+00 9.128018379781962039e+02 -3.018376768179616310e+00 -3.357093551886409788e-01 -3.619571838280575271e+00 -2.143989993963849816e+00 3.083424130001412994e-02 -8.445021012582607511e-01 2.424009713360984197e-01 1.477877156964668703e-02 -4.180196827706635254e+00 -3.151673411095621713e+00 -2.595616593910985738e-03 -4.819572335036378163e+00 -2.431462886786474265e+00 -2.010624171655670089e+00 -5.961524410720463685e+00 -2.305726776605871997e+00 -6.802382061491077714e+00 -1.988696607337682831e+00
-8.072867718038600060e+01 -3.018376768179616310e+00 8.942760568665043763e+01 9.916836562147740608e+01 -3.767704822711833401e+01 -3.261700499596039293e+01 2.988346236050226779e+01 8.534627078992050997e+01 3.211290639231271626e+01 1.335992571558448638e+01 -2.749150713817164871e+01 4.308026203832704937e+01 4.976852578010570460e-01 -7.231857954243464803e+01 4.450107538730605228e+01 2.811224322944057263e+01 -7.656497235156022896e+01 -2.775355207530230928e+01 -7.359678366145760720e+01 4.369381397528751165e+01
2.469966596815416437e+01 -3.357093551886409788e-01 9.916836562147736345e+01 -4.202158222617129013e+02 3.309644180747528708e+01 -3.542371515734818566e+00 6.615395817113427768e+01 2.192760753004992296e+02 6.428055432836158900e+01 2.954295246951267373e+01 7.422721262373191564e+01 1.993588164937004876e+02 1.202901873076485773e+00 -5.709765151529410332e+00 1.791186872602693825e+02 1.285953971174693891e+02 2.190242335481026714e+01 1.264962941884484238e+01 5.593140513873385089e+01 1.628984625862577218e+02
-1.097839013241607944e+02 -3.619571838280575271e+00 -3.767704822711832691e+01 3.309644180747528708e+01 4.594523752574601758e+02 -3.914648826141770144e+01 1.162799967255565470e+01 2.151874282298438956e+01 1.511883263396934929e+01 5.210959011955977438e+00 -6.274396811503735449e+01 -2.330309286485764630e+01 1.546794194191623006e-01 -8.762198681312392523e+01 -1.374016822750709821e+01 -1.462969835882552161e+01 -1.035347157544186274e+02 -3.934447716875323664e+01 -1.129590131672645015e+02 -8.493391784211262063e+00
-6.852810741588938015e+01 -2.143989993963849372e+00 -3.261700499596039293e+01 -3.542371515734818566e+00 -3.914648826141768723e+01 8.878033943417159435e+02 3.589350248202420879e-01 -9.052421697332807327e+00 2.647100485847968532e+00 1.712072805546870635e-01 -4.519597875980278445e+01 -3.402071615310268271e+01 -2.762036788019106917e-02 -5.214383842335318064e+01 -2.623670837338033834e+01 -2.170312665993464663e+01 -6.448772938145464195e+01 -2.494018803827587405e+01 -7.357170245076630977e+01 -2.145275462681359357e+01
1.097972658845432292e+01 3.083424130001417157e-02 2.988346236050227489e+01 6.615395817113427768e+01 1.162799967255565114e+01 3.589350248202426985e-01 -7.393601067799588691e+02 6.238408603776006345e+01 1.797746405187277574e+01 8.323541308229419400e+00 2.358449424184674825e+01 5.822767685267401561e+01 3.409354003691761337e-01 1.442431327297973631e+00 5.206252057627619934e+01 3.754531322210371513e+01 9.954104873659773034e+00 5.028511749220012206e+00 2.008497639176228233e+01 4.720696648642023519e+01
6.370785951675754788e+00 -8.445021012582607511e-01 8.534627078992050997e+01 2.192760753004992296e+02 2.151874282298438601e+01 -9.052421697332814432e+00 6.238408603776006345e+01 -3.357437803265510183e+02 6.118384923293453426e+01 2.786209746848519231e+01 5.875976590832684110e+01 1.793473299092635216e+02 1.125937583110405926e+00 -1.822726392144243590e+01 1.622059939771909569e+02 1.157466700919421498e+02 4.732413343190534860e+00 5.765510706954180442e+00 3.453801984940604086e+01 1.481107638250558409e+02
1.740591824716803870e+01 2.424009713360984752e-01 3.211290639231271626e+01 6.428055432836157479e+01 1.511883263396934929e+01 2.647100485847966755e+00 1.797746405187277574e+01 6.118384923293452715e+01 -9.608895889168226176e+02 8.026653691070530172e+00 2.727312856113802653e+01 5.964313616019694564e+01 3.322090782134056175e-01 6.564643364082887622e+00 5.291342381835914210e+01 3.843485013997858601e+01 1.601413246698908566e+01 7.332543383086719047e+00 2.670510597315474044e+01 4.774676657649889933e+01
4.935306545000330836e+00 1.477877156964663846e-02 1.335992571558448461e+01 2.954295246951267373e+01 5.210959011955977438e+00 1.712072805546870913e-01 8.323541308229419400e+00 2.786209746848519231e+01 8.026653691070530172e+00 -7.562831737725715584e+02 1.055302279011132072e+01 2.601779611046886131e+01 1.522589173222521908e-01 6.686780086450638816e-01 2.326105815243021979e+01 1.677621287133347039e+01 4.475408870823499363e+00 2.257244380228041081e+00 9.003663185657392987e+00 2.109054045233117591e+01
-1.213444492077768189e+02 -4.180196827706635254e+00 -2.749150713817164871e+01 7.422721262373190143e+01 -6.274396811503735449e+01 -4.519597875980278445e+01 2.358449424184674470e+01 5.875976590832684110e+01 2.727312856113802653e+01 1.055302279011132427e+01 2.590296141228220108e+02 4.536135074764947817e+00 3.641193820842948958e-01 -1.008163084424350728e+02 1.228321093179118861e+01 3.384279087560759791e+00 -1.146543247239546730e+02 -4.289489699631642594e+01 -1.200914648794531416e+02 1.573542807910867580e+01
-6.980803836514641603e+01 -3.151673411095621269e+00 4.308026203832704937e+01 1.993588164937005160e+02 -2.330309286485764630e+01 -3.402071615310268982e+01 5.822767685267401561e+01 1.793473299092635216e+02 5.964313616019694564e+01 2.601779611046886131e+01 4.536135074764954034e+00 -3.333006622195796353e+01 1.013246611390785423e+00 -7.450854768937065842e+01 1.213798525145471388e+02 8.332023882291510120e+01 -6.686279133952474751e+01 -2.221065371317468617e+01 -4.926899449982362000e+01 1.135989275790304447e+02
9.955283468031898841e-02 -2.595616593910985304e-03 4.976852578010569350e-01 1.202901873076485995e+00 1.546794194191623006e-01 -2.762036788019106223e-02 3.409354003691761337e-01 1.125937583110405704e+00 3.322090782134056175e-01 1.522589173222521630e-01 3.641193820842948958e-01 1.013246611390785423e+00 -6.179938144417375270e+02 -5.048095179835515589e-02 9.121250244197667278e-01 6.536872842958579843e-01 8.677531332991893220e-02 5.508785558217372397e-02 2.584045987707585534e-01 8.304997833400887819e-01
-1.537067629831232978e+02 -4.819572335036379052e+00 -7.231857954243464803e+01 -5.709765151529394345e+00 -8.762198681312392523e+01 -5.214383842335318064e+01 1.442431327297976740e+00 -1.822726392144243590e+01 6.564643364082889399e+00 6.686780086450638816e-01 -1.008163084424350870e+02 -7.450854768937065842e+01 -5.048095179835505875e-02 7.718071070683939752e+02 -5.721686291162151861e+01 -4.751825306900892798e+01 -1.446571941995194095e+02 -5.590499647645498982e+01 -1.647365855193095570e+02 -4.662596520128344224e+01
-5.002806584102325616e+01 -2.431462886786474265e+00 4.450107538730606649e+01 1.791186872602694109e+02 -1.374016822750709643e+01 -2.623670837338033124e+01 5.206252057627618512e+01 1.622059939771909569e+02 5.291342381835913500e+01 2.326105815243021979e+01 1.228321093179119572e+01 1.213798525145471388e+02 9.121250244197667278e-01 -5.721686291162151861e+01 -1.025626342706504204e+02 7.854005698316768758e+01 -4.812640464939676832e+01 -1.534665187398932495e+01 -3.072311026642314502e+01 1.056012400681237864e+02
-4.432063532967828223e+01 -2.010624171655670978e+00 2.811224322944057263e+01 1.285953971174693891e+02 -1.462969835882552161e+01 -2.170312665993465373e+01 3.754531322210370803e+01 1.157466700919421640e+02 3.843485013997857891e+01 1.677621287133346684e+01 3.384279087560759791e+00 8.332023882291510120e+01 6.536872842958578733e-01 -4.751825306900893509e+01 7.854005698316768758e+01 -1.120492858302675074e+02 -4.246239112151878459e+01 -1.406952639420813966e+01 -3.102450469281572865e+01 7.347382112908364604e+01
-1.857468719893926448e+02 -5.961524410720463685e+00 -7.656497235156022896e+01 2.190242335481026714e+01 -1.035347157544186132e+02 -6.448772938145464195e+01 9.954104873659773034e+00 4.732413343190536636e+00 1.601413246698907855e+01 4.475408870823501140e+00 -1.146543247239546730e+02 -6.686279133952474751e+01 8.677531332991897384e-02 -1.446571941995194095e+02 -4.812640464939678253e+01 -4.246239112151877038e+01 5.160229604949813620e+02 -6.710492762144528456e+01 -1.954321457217347984e+02 -3.712355645944076343e+01
-7.120919595547842107e+01 -2.305726776605871997e+00 -2.775355207530230928e+01 1.264962941884485659e+01 -3.934447716875324375e+01 -2.494018803827587405e+01 5.028511749220012206e+00 5.765510706954178666e+00 7.332543383086717270e+00 2.257244380228041081e+00 -4.289489699631642594e+01 -2.221065371317467907e+01 5.508785558217375866e-02 -5.590499647645498982e+01 -1.534665187398932495e+01 -1.406952639420813611e+01 -6.710492762144527035e+01 6.033411495420373285e+02 -7.438417796089414935e+01 -1.139368047502503423e+01
-2.072681875815117678e+02 -6.802382061491077714e+00 -7.359678366145760720e+01 5.593140513873387931e+01 -1.129590131672645015e+02 -7.357170245076630977e+01 2.008497639176228589e+01 3.453801984940604086e+01 2.670510597315474044e+01 9.003663185657396539e+00 -1.200914648794531558e+02 -4.926899449982363421e+01 2.584045987707584424e-01 -1.647365855193095570e+02 -3.072311026642316278e+01 -3.102450469281574286e+01 -1.954321457217347984e+02 -7.438417796089414935e+01 3.309082431680460559e+02 -2.040879177224442032e+01
-3.843893266686813348e+01 -1.988696607337682831e+00 4.369381397528751165e+01 1.628984625862577502e+02 -8.493391784211262063e+00 -2.145275462681358647e+01 4.720696648642022097e+01 1.481107638250558693e+02 4.774676657649889222e+01 2.109054045233117591e+01 1.573542807910867580e+01 1.135989275790304447e+02 8.304997833400887819e-01 -4.662596520128343514e+01 1.056012400681237864e+02 7.347382112908363183e+01 -3.712355645944074922e+01 -1.139368047502503067e+01 -2.040879177224442032e+01 -1.539953061031469588e+02
