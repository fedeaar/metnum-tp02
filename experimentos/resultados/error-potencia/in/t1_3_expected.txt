-9.780000000000000000e+02
9.130000000000000000e+02
9.110000000000000000e+02
8.890000000000000000e+02
-7.600000000000000000e+02
-7.580000000000000000e+02
6.910000000000000000e+02
6.830000000000000000e+02
-6.540000000000000000e+02
6.290000000000000000e+02
-6.180000000000000000e+02
5.450000000000000000e+02
-5.400000000000000000e+02
5.190000000000000000e+02
3.190000000000000000e+02
-2.520000000000000000e+02
-2.160000000000000000e+02
-1.660000000000000000e+02
-1.620000000000000000e+02
8.900000000000000000e+01
